File: 1672. Richest Customer Wealth/Richest_Customer_Wealth.java
public class Richest_Customer_Wealth {
    public static int maximumWealth(int[][] accounts) {
        int max = 0;
        for (int i[] : accounts) {
            int total = 0;
            for (int j : i) {
                total += j;
            }
            if (total > max) {
                max = total;
            }
        }
        return max;
    }
    public static void main(String[] args) {
        int[][] accounts = {{1,2,3},{3,2,1}};
        int result = maximumWealth(accounts);
        System.out.print(result);
    }
}

File: 1672. Richest Customer Wealth/Richest_Customer_Wealth.py
class Solution:
    def maximumWealth(self, accounts: List[List[int]]) -> int:
        return max(sum(customer) for customer in accounts)

accounts = [[1,5],[7,3],[3,5]]
print(Solution().maximumWealth(accounts))

# nếu sử dụng các hàm của py thì việc compile code sẽ có vẻ nhanh hơn